src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,95 @@
-import { PeopleFilters } from './PeopleFilters';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { useEffect, useMemo, useState } from 'react';
 import { Loader } from './Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
 import { PeopleTable } from './PeopleTable';
+import { PeopleFilters } from './PeopleFilters';
+import { filterPeople } from '../utils/filterPeople';
 
 export const PeoplePage = () => {
+  const { slug } = useParams();
+  const [people, setPeople] = useState<Person[]>([]);
+  const [
+    selectedPersonSlug,
+    setSelectedPersonSlug,
+  ] = useState<string>(slug || '');
+  const [isError, setIsError] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [searchPearams] = useSearchParams();
```

Comment:
There is a typo in the variable name. It should be 'searchParams', not 'searchPearams'.